wiki/Ranking_criteria/Metadata/en.md

```diff
@@ -216,10 +217,9 @@ When the entire field is uppercase or lowercase, markers may use alternative cas
 
 ## Romanisation
 
-This section covers what to do when romanising metadata yourself. If you use an official translation or romanisation, you must use it as-is in the romanised fields instead.
-
 ### Rules
 
+- **Romanisation rules only apply when you romanise the metadata yourself. If you use an official translation or romanisation, you must use it as-is in the romanised fields instead.**
```

Comment:
this seems wrong though. this line is supposed to apply to the entire "Romanisation" section, not just "Romanisation > Rules"

if someone missed that then that's their fault imo. RC was fine here already